### ISSUE TITLE
Export of certain sat files stop mid export

### ISSUE THREAD

**Issue by Laythros** [philm001/SWGModelExporter#7]
Morning philm, not sure if you are still checking or supporting this but would post it here so if future people run into the same issue.

Exporting SATs as a whole works great, they come into FBX, the animations seem to be rotated -90 on the x axis, but thats an easy fix.

Issue is when I try to export out a few of the bigger .sat files it just stops mid processing like it hits a processing limit...

SATs I have tested on are:
4lom.sat
ackbar.sat
Stormtrooper.sat

Thanks.

**Comment by philm001** [comment]
Hello 

Thank you for posting! Glad you are using it. I haven't had much time to refine the tool these days due to a very busy schedule. Please though, if you fix any bugs, feel free to submit a PR.

As for the bigger SATs, have you tried waiting? I know that for some of the big ones, it takes awhile to export. I was waiting for 5 or 10 min or even longer for those. I wanted to try to multiple thread the export but haven't figured out a way around it.

**Comment by Laythros** [comment]
I got back to the process after work, and it did indeed complete for Stormtrooper and 4Lom but not ackbar....

So to update, if it looks like it stopped processing in the commandline its probably still running, and this ended up as a 500 MB file at the end....

**Comment by philm001** [comment]
Yeah, the character models are huge because it is calculating all of the animations for each of the components. a little clunky. But I do remember it working with ackbar because that is what I was testing with!

I do believe that there are a number of improvements that can be made to the code to reduce code size. 

Also, this will export ALL of the animations associated with the skeleton. So this means that ackbar can animate according to different gender animations and also to jedi specific animations, it is quite interesting.
